docs/configuration.rst

```diff
@@ -0,0 +1,137 @@
+Configuration
+=============
+
+Optuna Dashboard supports TOML configuration files for managing complex settings that would be cumbersome to specify via command-line arguments.
+
+Basic Usage
+-----------
+
+.. code-block:: console
+
+    $ optuna-dashboard --from-config config.toml
+
+Configuration File Structure
+----------------------------
+
+Dashboard Settings
+~~~~~~~~~~~~~~~~~~
+
+Settings that can be specified via command-line options can also be configured in TOML files.
+Configuration file settings are overridden by command-line arguments.
+
+.. code-block:: toml
+
+    [optuna_dashboard]
+    storage = "sqlite:///example.db"
+    storage_class = "RDBStorage"
+    port = 8080
+    host = "127.0.0.1"
+    server = "auto"
+    artifact_dir = "/path/to/artifacts"
```

Comment:
I've noticed that there are two ways to enable FileSystemArtifact.

```
[optuna_dashboard]
artifact_dir = "/path/to/artifacts"

[artifact_store.filesystem]
base_path = "/path/to/artifacts"
```

In my opinion, we should avoid providing the `optuna_dashboard.artifact_dir` option, since the latter approach is more consistent and flexible. How about removing this line from the documentation? Later, we can also update the implementation in a future pull request:

https://github.com/optuna/optuna-dashboard/blob/1ecfa1b1c37a7c91da41206dd4c3aec8eaa1aab3/optuna_dashboard/_cli.py#L137

```suggestion
```

Reply:
Thank you for pointing it out! Sounds reasonable.